data-hibernate-jpa/src/test/java/io/micronaut/data/hibernate/entities/EventTest.java

```diff
@@ -55,6 +64,11 @@ public void setDateUpdated(LocalDateTime dateUpdated) {
         this.dateUpdated = dateUpdated;
     }
 
+    public void setValue(int value) { this.value = value; }
+
+    public int getValue() { return value; }
+
+    @Transient
```

Comment:
`@Transient` was missing from `prePersist` which broke some test behavior. This was getting persisted and was thus passing from the saved entity to the updated entity.